### ISSUE TITLE
Freeform Rows have quotations around the class name—breaking CSS styling

### ISSUE THREAD

**Issue by vonscriptor** [solspace/craft3-freeform#1258]
**Describe the bug or issue you're experiencing**

It seems that Freeform is rendering the form with double quotes around `freeform-row` element's class names. This is breaking the CSS styling for those elements.

`class=""freeform-row""`

**Steps to reproduce**

1. Render a form that uses a freeform row
2. See that the class name has quotations as part of the name

**Expected behavior**
The freeform-row class name should not have quotation marks surrounding it.

**Craft & Plugin Info (please complete the following information):**

- Craft Version: Pro 4.9.1
- Freeform Version: 5.1.17
- Freeform Edition: Pro
- Fresh Install or Upgrade: Fresh

**Comment by kjmartens** [comment]
Sorry about this @vonscriptor,

I have just noticed this issue as well. I'm not sure why it's doing that now and when it started. I don't see any changes to the code throughout v5.0. 🤷‍♂️  Anyway, we'll check into it on Monday. In the meantime, it may be worth hardcoding `<div class="freeform-row">` instead of `<div{{ form.attributes.row }}>`.

**Comment by darylknight** [comment]
This is affecting the "Bootstrap 5" template too.

**Comment by kjmartens** [comment]
@vonscriptor @darylknight,

It is very weird that this just started happening recently, and apparently, there isn't a way to make it work as is. In the next release of Freeform, we will update all sample formatting templates to include the `|raw` filter:

``` diff
- <div{{ form.attributes.row }}>
+ <div{{ form.attributes.row|raw }}>
```

We'll update the documentation shortly as well.

**Comment by kjmartens** [comment]
We've released Freeform 5.1.18 now, which updates the sample formatting templates to include the `|raw` filter. If you have any custom formatting templates using `form.attributes.row`, please ensure you're adding the `|raw` filter to them as well. 🙂

**Comment by vonscriptor** [comment]
This is also happening on `freeform-form-errors`:

`<div class=""freeform-form-errors"">`

**Comment by kjmartens** [comment]
I'm very sorry about this @vonscriptor. I meant to check that as well, but it slipped my mind. I have fixed it now and released Freeform 5.1.18.1. 🙂 

Same thing goes for these if you're using them in a custom formatting template:

``` diff
- <div{{ form.attributes.success }}>
+ <div{{ form.attributes.success|raw }}>
```

``` diff
- <div{{ form.attributes.errors }}>
+ <div{{ form.attributes.errors|raw }}>
```

**Comment by vonscriptor** [comment]
Thanks! It appears to be working well now. I appreciate the quick fixes for this!

For anyone else seeing this, Utilities > Caches need to be cleared for this to take effect.
